src/containers/UsernameTranslation.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+
+/*
+At the moment works ONLY with apiPath='seacat_auth'
+- userId (required, array) - user ids to be translated
+- apiPath - default (and only working so far) 'seacat_auth'
+- cleanupTime - default set to 24 hours
+
+Usage:
+
+import { Credentials } from 'asab-webui';
+...
+<Credentials app={props.app} id={['user1-id', 'user2-id', 'user3-id']} apiPath='seacat_auth' cleanupTime={1000 * 60 * 60 * 24 * 14}/>
+
+*/
+
+export function Credentials(props) {
+
+	const { t } = useTranslation();
+
+	const apiPath = props.apiPath ?? 'seacat_auth';
+	let API = props.app.axiosCreate(apiPath);
+
+	const userIdsArray = Array.isArray(props.userId) ? props.userId : [props.userId] ;
+
+	const cleanupTime = props.cleanupTime ?? 1000 * 60 * 60 * 24; // 24 hrs
+
+	const [usernames, setUsernames] = useState([]);
```

Comment:
@petrKavulok if the component is named as `Credentials`, it might be also good to rename `usernames` everywhere in the code to `credentials` (also the names of the consts)